### ISSUE TITLE
replay +10' not working

### ISSUE THREAD

**Issue by fronzee** [XCSoar/XCSoar#933]
XCSoar versions having and not having the problem
-------------------------------------------------
all versions down to 6.6 not working (PC versions) and 7.23 android not working. 

Expected behavior
-----------------

when opening replay and pressing +10' it should advance log file 10min ahead. 

Actual behavior
---------------

Does nothing. 

Steps to reproduce the behavior
-------------------------------

File recorded during actual flight with android 7.23 version xcsoar.
choose file to replay - press +10'

Do you have any idea what may have caused this?
-----------------------------------------------
- something changed in igc file format during the years that causes it?

Do you have an idea how to solve the issue?
-------------------------------------------

**Comment by lordfolken** [comment]
Can you attach the igc file?

**Comment by fronzee** [comment]
[2022-05-23-XCS-XYA-01.zip](https://github.com/XCSoar/XCSoar/files/8765851/2022-05-23-XCS-XYA-01.zip)
igc itself not allowed to upload, so compressed attached

**Comment by lordfolken** [comment]
Seems to happen with any igc file i tried.

**Comment by fronzee** [comment]
In addition to +10' I would say -10' would be good to have too, in case you go 10' too much in a 5hours flight... There would be a nice chance to just go back 10' instead of doing it all again from the beginning.

**Comment by lordfolken** [comment]
Yeah the function needs work. There is also a demo mode if you leave the filename empty and press play. Very intuitive. 🙃

**Comment by kobedegeest** [comment]
@coderabbitai have a look and propose a fix

**Comment by lordfolken** [comment]
**Open PR:** #2472 — [mockup] Replay: flight-minute seek (experimental).

**Comment by WolfgangDrescher** [comment]
Actually, this `+10'` button does not skip 10 minutes, but instead enters a fast-forward mode where the flight is played faster for probably 10 flight minutes (I did not wait for the full cycle, so I can't say for sure).

Can we agree to change this behavior to skip 10 minutes, as we already have a rate feature to play back a flight faster?

I'm working on seek-to-next-cruise/circling buttons which could be a more intuitive way to navigate through a flight than jumping ahead by 10 minutes. See https://github.com/WolfgangDrescher/XCSoar/tree/replay-seek
